test/powershell/Modules/Microsoft.PowerShell.Utility/Tee-Object.Tests.ps1

```diff
@@ -92,4 +92,18 @@ Describe "Tee-Object DRT Unit Tests" -Tags "CI" {
         $results | Should -Be $expected
     }
 
+    It "Positive Variable/ErrorRecordVariable Test" {
+        $results = Get-Item $PSHOME,doesnotexist1,$PWD,doesnotexist2 2>&1 | Tee-Object -Variable outputVar -ErrorRecordVariable errVar
```

Comment:
I think you should have a test case with a native command to ensure that use case doesn't break